src/runners/alert_processor.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python
+
+import json
+import uuid
+
+from .config import ALERTS_TABLE, DATABASE
+from .helpers import db, log
+
+# After alerts are created but before they get turned into tickets, they get processed; this processing step is where logic like alert grouping is applied.
```

Comment:
Ah, this was one of the comments that I wrote when it was an empty document and I wanted to record what's supposed to happen so that I have a clear goal in my mind while I'm coding. Removed.